build-system/tasks/runtime-test/helpers.js

```diff
@@ -192,10 +192,14 @@ function unitTestsToRun(unitTestPaths) {
   }
 
   filesChanged.forEach(file => {
+    if (!fs.existsSync(file)) {
+      log(yellow('SKIP:'), 'Skipping removed file', file);
```

Comment:
This change works. For consistency though, change the logging statement to:
```
if (!process.env.TRAVIS) {
  log(green('INFO:'), 'Skipping', cyan(file), 'because it was deleted');
}
```